setup.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+from __future__ import absolute_import, print_function
+import os
+
+from setuptools import setup, find_packages
+
+NAME = 'ft'
```

Comment:
`ft` is nice and short - but it's also hard to find. I'd prefer to call it `shell-functools` for now (unless someone has a better suggestion for a name)

Reply:
Both names are not taken. https://pypi.python.org/pypi?%3Aaction=search&term=shell-functools&submit=search. I am okay with either personally